pyproject.toml

```diff
@@ -8,6 +8,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.11"
 typer = "^0.12.3"
+click = "<8.2.0"
```

Comment:
Let's pin the typer instead to [0.15.4](https://github.com/fastapi/typer/releases/tag/0.15.4)